sci-libs/rocBLAS/rocBLAS-4.1.0.ebuild

```diff
@@ -0,0 +1,110 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="AMD's library for BLAS on ROCm."
+HOMEPAGE="https://github.com/ROCmSoftwarePlatform/rocBLAS"
+SRC_URI="https://github.com/ROCmSoftwarePlatform/rocBLAS/archive/rocm-${PV}.tar.gz -> rocm-${PN}-${PV}.tar.gz
+	https://github.com/ROCmSoftwarePlatform/Tensile/archive/rocm-${PV}.tar.gz -> rocm-Tensile-${PV}.tar.gz"
+
+LICENSE="MIT"
+KEYWORDS="~amd64"
+IUSE="benchmarks tests"
+SLOT="0"
+
+RDEPEND="=dev-util/hip-$(ver_cut 1-2)*"
+DEPEND="${RDEPEND}
+	dev-perl/File-Which
+	dev-libs/msgpack
+	dev-util/cmake
+	dev-util/rocm-cmake
+	!dev-util/Tensile
```

Comment:
It doesn't hurt to have Tensile installed on the system, right?  We can alleviate this requirement.

Reply:
System installed Tensile polluted the one in virtualenv, for some unknown reasons,  and if the mismatch of version will cause build failure

Reply:
OK, let's keep the blocker.